.github/workflows/build_and_push_image_on_tag.yml

```diff
@@ -0,0 +1,40 @@
+# This is a basic workflow to help you get started with Actions
+
+name: "Build and push on tag event"
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on tag events events but only for the master branch
+  push:
+    tags:
+      - '*'
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
```

Comment:
This is a really small nitpick, but we should probably try and run the tests on the same operating system which our Jenkins instances run on.

There's a very small chance it'll impact anything, but just in case

Reply:
NVM, not relevant for this job